aws-production-2/main.tf

```diff
@@ -261,6 +270,53 @@ module "aws_asg_org" {
   ]
 }
 
+module "aws_asg_org_canary" {
+  source                     = "../modules/aws_asg_canary"
+  cyclist_auth_token         = "${random_id.cyclist_token_org.hex}"
```

Comment:
Do we still need to connect this to cyclist, since it won't be autoscaled?

Reply:
Oh, never mind, I just noticed that it's not actually set further down. 😅

Reply:
Cyclist is still needed to perform a graceful manual scale-in. We will manually bump the desired instances on the ASG. In the case of lowering desired instances (e.g. setting it to 0) we need cyclist to manage the lifecycle hooks, otherwise the instance will be terminated before the jobs are completed, which could impact production jobs running on the instance.